src/sentry/replays/usecases/ingest/dom_index.py

```diff
@@ -396,6 +396,15 @@ def _handle_breadcrumb(
 ) -> ReplayActionsEventPayloadClick | None:
 
     click = None
+    project = Project.objects.get(id=project_id)
+    dead_click_timeout = (
+        5000
```

Comment:
you could also make this an option if you wanted to have it be more controllable. but this makes sense!